src/code_805D8C8_1.c

```diff
@@ -2867,6 +1780,17 @@ u16 GetLeaderActionId(void)
     return GetLeaderInfo()->action.action;
 }
 
+// Could this be a start of a new file?
```

Comment:
50/50 chance. Usually the `pksdir0` has been the divider but I wouldn't be surprised if this is a new file either.